src/views/artefact-editor/roi-layer.vue

```diff
@@ -5,34 +5,45 @@
             :key="roi.id"
             :d="roi.shape.svg"
             :transform="`translate(${roi.position.x} ${roi.position.y})`"
-            :class="{ shine: roi.shiny, selected: roi === selected, highlighted: highlighted(roi) }"
+            :class="{ shine: roi.shiny, selected: isSelectedRoi(roi), highlighted: highlighted(roi) }"
             @click="onPathClicked(roi)"
             vector-effect="non-scaling-stroke"
         />
     </g>
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue, Emit } from 'vue-property-decorator';
+import { Component, Prop, Vue, Emit, Watch } from 'vue-property-decorator';
 import { InterpretationRoi, SignInterpretation } from '@/models/text';
+import { i } from 'mathjs';
```

Comment:
`i`, either I am really dense, or this is very unhelpful variable name (though apparently no fault of your own).